reference/5.1/Microsoft.PowerShell.Core/About/about_Functions_Advanced_Parameters.md

```diff
@@ -172,6 +172,23 @@ Param(
 )
 ```
 
+The boolean argument types default to **False** when omitted from the
+**Parameter** attribute. Set the argument value to `$true` or just list the
+argument by name. For example, the following **Parameter** attributes are
+equivalent.
```

Comment:
Hate to nitpick, but they don't _all_ default to `$false`. For example, `[CmdletBinding()]` has a `PositionalBinding` property that automatically defaults to `$true` and must explicitly be set to `$false` if you want to disable it.

But it tends to be the exception rather than the rule (I'm actually not even sure there _are_ other exceptions in the core PS library attributes). Point being, it defaults to whatever the attribute definition says the default is. 🙂

Reply:
Oh, Interesting. Thanks. I will fix that.

Reply:
Ok, this paragraph is specifically talking about the boolean parameter arguments. `PositionalBinding` is a `ParameterAttribute`. But I could see where people might get confused. I will make this clearer.

Reply:
Not sure what you mean there @sdwheeler.

`PositionalBinding` is a property of the `CmdletBindingAttribute` that. can be set via the attribute declaration, just like `Mandatory`; they just have different defaults if not specifically set in the attribute declaration. 🙂

Reply:
Oops, copy/paste wrong type in my last comment.

I updated the article to be a little more specific. All of the boolean arguments to the `ParameterAttribute` class default to **False**. And that paragraph is specifically addressing those arguements.

`PositionalBinding` is a property of the `CmdletBindingAttribute`, which defaults to **True**, as documented in this article and in about_Functions_CmdletBindingAttribute.

Reply:
Oh, I see what you were saying. No worries, and thanks for fixing it 😊 💖